Dockerfile

```diff
@@ -78,7 +81,10 @@ ENV PGROOT=/var/lib/postgresql \
     OR_REINDEX_COUNTER=${OR_REINDEX_COUNTER} \
     OR_DISABLE_REINDEX=${OR_DISABLE_REINDEX:-false} \
     POSTGRES_MAX_CONNECTIONS=${POSTGRES_MAX_CONNECTIONS:-50} \
-    OR_DISABLE_AUTO_UPGRADE=${OR_DISABLE_AUTO_UPGRADE:-false}
+    OR_DISABLE_AUTO_UPGRADE=${OR_DISABLE_AUTO_UPGRADE:-false} \
+    # VACUUM FULL settings (disabled by default)
+    OR_ENABLE_VACUUM_FULL=${OR_ENABLE_VACUUM_FULL:-false} \
+    OR_VACUUM_FULL_CRON_SCHEDULE="${OR_VACUUM_FULL_CRON_SCHEDULE:-'0 0 * * 0'}"
```

Comment:
Don't think we need two vars for this `OR_VACUUM_FULL_CRON_SCHEDULE` should be sufficient if not set then don't create and/or remove cronjob